controllers/propertiesController.js

```diff
@@ -7,7 +8,7 @@ getAllProperties = async (req, res) => {
     res.send(properties);
   };
 
-createProperty = (req, res, next) => {
+createProperty = auth,(req, res, next) => {
```

Comment:
same comment. Ensure this pattern is removed from all routes where you added them